Documentation/troubleshooting.rst

```diff
@@ -16,6 +16,57 @@ We use GitHub issues to maintain a list of `Cilium Frequently Asked Questions
 (FAQ)`_. You can also check there to see if your question(s) is already
 addressed.
 
+Overall Health
+==============
+
+The first step in troubleshooting is to retrieve an overview of the overall
+health. This is achieved by running the ``cilium status`` command.
+
+Kubernetes
+----------
+
+When using Kubernetes, refer o the ``k8s-cilium-exec.sh`` script to execute
+``cilium status`` on all cluster nodes with ease. Download the
+``k8s-cilium-exec.sh`` script:
+
+.. code:: bash
+
+    $ curl -sLO releases.cilium.io/v1.0.0-rc4/tools/k8s-cilium-exec.sh
```

Comment:
Is the `v1.0.0-rc4` supposed to be hardcoded?

Reply:
I don't have a better idea yet as we won't have a branch for any tagged release until 1.0 goes out